README.md

```diff
@@ -9,7 +9,7 @@ Currently it gets activated once you open a file with `.sw` extension.
 
 ## Testing
 
-Prior to testing make sure you have [`forc`](https://github.com/FuelLabs/sway/tree/master/forc) installed as well.
+Prior to testing make sure you have [`forc`](https://fuellabs.github.io/sway/latest/introduction/installation.html) installed as well.
```

Comment:
Technically, a while back `forc lsp` was turned into its own binary (https://github.com/FuelLabs/sway/pull/1179), so you need both `forc` and `forc-lsp` installed. Might as well say the entire Fuel toolchain needs to be installed to avoid any issues.

```suggestion
Prior to testing make sure you have [the Fuel toolchain](https://fuellabs.github.io/sway/latest/introduction/installation.html) installed as well.
```